### DISCUSSION TITLE
Shoji-Nishiyama variants

### DISCUSSION THREAD

**Discussion by lebraunjames** [mtex-toolbox/mtex#1832] [Ask Anything] [answered]
I'm trying to check the variants of the Shoji-Nishiyama OR:
  (111)γ-fcc || (0001)ε-hcp
  [1¯10]γ-fcc || [11¯20]ε-hcp

Here is my input:
```matlab
mori = orientation.map(Miller(1, 1, 1,cs_fcc),Miller(0, 0, 0, 1,cs_hcp),...
      Miller(1, -1, 0,cs_fcc,'uvw'),Miller(1, 1, -2, 0,cs_hcp,'uvw'));
```
However, when i check the variants of this relationship:
```matlab
round2Miller(mori.variants)
```
The answer is this:
```
    plane parallel   direction parallel    fit
  (111) || (0001)    [10-1] || [010]       0°
 (1-11) || (0001)     [110] || [100]       0°
(1-1-1) || (000-1)   [01-1] || [010]       0°
 (11-1) || (0001)    [1-10] || [010]       0°
```
instead of [11-20] or [110] for the hcp, it uses the <100> direction. Is there something i am missing that makes the direction for hcp wrong?

**Comment by ralfHielscher** [comment, marked as answer]
Hi James,

this is a small bug in MTEX. Thank you for pointing this out. We will fix it shortly. 

Also, you do not have to worry about it. Your misorientation variants are perfectly defined and you can go on and work with them. Its only an issue of the display. You can checkout the right display with
```matlab
round2Miller(mori.variants(1))
round2Miller(mori.variants(2))
round2Miller(mori.variants(3))
```
Ralf.

**Reply by lebraunjames** [reply]
Thank you! I tried doing it this way and i can now confirm that the variants are correct.
